test/grid/a11y-spec.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import Enzyme, { mount } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import assert from 'power-assert';
+import Grid from '../../src/grid';
+import { unmount, testReact } from '../util/a11y/validate';
+
+/* eslint-disable react/jsx-filename-extension */
+/* global describe it beforeEach afterEach */
+
+const { Row, Col } = Grid;
+
+Enzyme.configure({ adapter: new Adapter() });
+
+describe('Row', () => {
+    let wrapper;
+
+    afterEach(() => {
+        if (wrapper) {
+            wrapper.unmount();
+            wrapper = null;
+        }
+        unmount();
+    });
+
+    it('should simply pass', async () => {
```

Comment:
change to 'should not have any violations'

Reply:
Accepted